composer.json

```diff
@@ -140,16 +140,14 @@
 
     "autoload": {
         "psr-4": {
-            "HWI\\Bundle\\OAuthBundle\\": ""
-        },
-        "exclude-from-classmap": [
-            "/Tests/"
-        ]
+            "HWI\\Bundle\\OAuthBundle\\": "src/"
+        }
     },
 
     "autoload-dev": {
         "psr-4": {
-            "HWI\\Bundle\\OAuthBundle\\Tests\\": "Tests/"
+            "HWI\\Bundle\\OAuthBundle\\Tests\\App\\": "tests/App/",
```

Comment:
is this one needed? Seems it would be covered by the one below?

Reply:
Yes, and no... in theory it works, but i.e. in phpstorm it moans it's wrong when you open that class... I tried the approach from EasyAdmin, but results is same so I guess I can revert & schedule re-work of this integration/functional tests into something more "clear".